src/librustdoc/lib.rs

```diff
@@ -25,6 +25,7 @@ extern crate getopts;
 extern crate env_logger;
 extern crate rustc;
 extern crate rustc_data_structures;
+extern crate rustc_index;
```

Comment:
@eddyb  This works now, but previous 2018-edition-style(add rustc_index inside Cargo.toml then `rustc_index::xxx`) failed unexpectedly, did I miss something?

Reply:
Aah, if you add it to Cargo.toml it creates a copy - so make sure you only add it to Cargo.toml if rustc_data_structures is already in Cargo.toml.